release-notes/release-notes-2-4.md

```diff
@@ -1,29 +1,33 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Release 2.4 (Planned October 2020)
+# Release 2.4 (October 2020)
 
-Release 2.4 adds support for:
-   * Project management of IT change
-   * IoT and large scale cohorts
+* The [Integration Daemon](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/concepts/integration-daemon.md) now makes is simpler to exchange metadata with third party technology 
+  such as catalogs and databases.
+* The Repository explorer User Interface ('REX') can now be configured with a list of available 
+  platforms and servers to improve security and usability. 
+* The latest version of Repository Explorer, Type Explorer and Dino, is found 
+  in the 'Presentation Server' UI. See 
+  [Presentation Server component documentation](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/user-interfaces/presentation-server) 
+  and 
+  [Configuring the Presentation Server](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/user/configuring-the-presentation-server.md)
+  .
+* The [Business UI ](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring)
```

Comment:
I do not think Business UI is the right term for this . The Glossary Author is a "Business UI" in the Presentation server. I suggest Web Component UI

Reply:
I'm going to counter propose staying with the 'UI Chassis'. I agree Business UI is confusing for end user, but Web Component UI could be either - what is it.

We need to clarify the role of the UIs, but that's a longer task. For now I think staying with UI chassis makes sense, as that at least is already used, linked, and referred to in the broader docs & code.